src/main/java/org/folio/auth/authtokenmodule/tokens/TokenValidationContext.java

```diff
@@ -63,17 +79,15 @@ public TokenValidationContext(HttpServerRequest httpServerRequest,
     this.tokenCreator = tokenCreator;
     this.tokenToValidate = tokenToValidate;
     this.userService = userService;
+    this.allowCrossTenantRequests = BooleanUtils.toBoolean(System.getProperty("allow.cross.tenant.requests"));
```

Comment:
if there are other system properties defined in mod-authtoken and there is a central place where they are all defined, it will be good to include this line in that central place if it exists.

Reply:
system properties are defined in places where they need to be used, there is no central place, as option to improve I can create class with constants for all system variable names

Reply:
if they are defined where needed, you can leave as is.